metadata.rb

```diff
@@ -1,14 +1,26 @@
-name 'perl'
-maintainer 'Chef Software, Inc.'
-maintainer_email 'cookbooks@chef.io'
-license 'Apache-2.0'
-description 'Installs perl and provides a resource for maintaining CPAN modules'
-version '7.0.1'
+name              'perl'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
+license           'Apache-2.0'
+description       'Installs perl and provides a resource for maintaining CPAN modules'
+version           '7.0.1'
+source_url        'https://github.com/sous-chefs/perl'
+issues_url        'https://github.com/sous-chefs/perl/issues'
+chef_version      '>= 13.4'
 
-%w(ubuntu debian linuxmint redhat centos amazon scientific oracle fedora windows zlinux suse opensuse opensuseleap).each do |os|
-  supports os
-end
-
-source_url 'https://github.com/chef-cookbooks/perl'
-issues_url 'https://github.com/chef-cookbooks/perl/issues'
-chef_version '>= 13.4'
+supports 'amazon'
+supports 'centos'
+supports 'debian'
+supports 'fedora'
+supports 'freebsd'
+supports 'linuxmint'
+supports 'opensuse'
+supports 'opensuseleap'
+supports 'oracle'
+supports 'redhat'
+supports 'scientific'
+supports 'smartos'
+supports 'suse'
+supports 'ubuntu'
+supports 'windows'
+supports 'zlinux'
```

Comment:
Should we only mark the platforms we test on as supported?

Reply:
It's kind of hard to test on zlinux and was already included. I'd rather keep it like it was and assume it works until someone says otherwise.

Reply:
zlinux is just centos so it's fine